### DISCUSSION TITLE
Forth extensions

### DISCUSSION THREAD

**Discussion by paaguti** [orbitalquark/scintillua#35] [General]
HI,
I've seen that Scintillua has dropped .fs as an extension for Forth. However, if you look at
https://www.complang.tuwien.ac.at/forth/gforth/Docs-html/Using-the-Cross-Compiler.html#Using-the-Cross-Compiler
you will see it used (at least 4 times)

Consider bringing it back...

Thanks, /PA

**Comment by orbitalquark** [comment]
There is a conflict for .fs between Forth and F#. I picked F# since it is a newer language. If Forth is using it more than F#, I can reverse this.
